README.md

```diff
@@ -22,6 +22,28 @@ Once you have Docker properly installed, follow these instructions:
 4. Open up the browser and connect to **`localhost:6080`** to get to the workspace desktop GUI.
 5. Once done, from the container shell press **CTRL+D**.
 
+## Instructions to test the skin
+Connect the `ESD USB/CAN` interface to the host machine, then :
+```sh
+    $ sudo ip link set can0 type can bitrate 1000000
+    $ sudo ip link set up can0
```

Comment:
Where are these instructions supposed to be run? In the host?
If so, better off saying that explicitly. 

If they need to be run in the containers instead, you don't need `sudo`.

Reply:
Also, if they can be run from within the container as well, it'd be probably better doing so.